### PR TITLE
FAQ Update #vac_cert_privileges_travel

### PR DESCRIPTION
**Pull request by dsarkar**
FAQ Update https://www.coronawarn.app/de/faq/#vac_cert_privileges_travel

### CODE REVIEW

**Comment by dsarkar** [comment]
Internal Tracking ID: [EXPOSUREAPP-8063](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-8063)

File: src/data/faq_de.json [hunk 1]
```diff
@@ -172,12 +172,15 @@
...
+                        "title": "Kann mein Zertifikat in der App als Berechtigung zum Reisen dienen?",
                         "anchor": "vac_cert_privileges_travel",
                         "active": true,
                         "textblock": [
-                            "Ob Sie mit einem vollständigen Impfschutz beispielsweise reisen dürfen, hängt von den Bestimmungen des jeweiligen Landes ab.",
-                            "Sie können unter anderem dieses <a href='https://reopen.europa.eu/de' target='_blank' rel='noopener noreferrer'>Webseite</a> benutzen, um ihre Reise innerhalb der EU zu planen."
+                            "Der digitale Impfnachweis kann nicht nur in Deutschland genutzt werden. Der in Deutschland ausgestellte digitale Impfnachweis kann auch im EU-Ausland als Nachweis für erfolgte Impfungen gelten, wenn der jeweilige EU-Mitgliedstaat digitale Nachweise zulässt. Dasselbe gilt perspektivisch auch für digitale Genesenenzertifikate und Testzertifikate. Im Allgemeinen prüfen die jeweiligen EU-Mitgliedstaaten  mit ihren eigenen Prüf-Apps Ihr Zertifikat, d. h. es ist unerheblich, in welcher Sprache Ihr Zertifikat auf Ihrem Gerät dargestellt wird.",
```
Comment (Ein-Tim) [inline src/data/faq_de.json:179]: Empty space to much

```suggestion
                            "Der digitale Impfnachweis kann nicht nur in Deutschland genutzt werden. Der in Deutschland ausgestellte digitale Impfnachweis kann auch im EU-Ausland als Nachweis für erfolgte Impfungen gelten, wenn der jeweilige EU-Mitgliedstaat digitale Nachweise zulässt. Dasselbe gilt perspektivisch auch für digitale Genesenenzertifikate und Testzertifikate. Im Allgemeinen prüfen die jeweiligen EU-Mitgliedstaaten mit ihren eigenen Prüf-Apps Ihr Zertifikat, d. h. es ist unerheblich, in welcher Sprache Ihr Zertifikat auf Ihrem Gerät dargestellt wird.",
```

**Review by Ein-Tim — CHANGES_REQUESTED** [review]
Please remove an empty space too much 🙂
Besides of this: LGTM!

**Comment by Ein-Tim** [comment]
English will follow separately?

File: src/data/faq_de.json [hunk 2]
```diff
@@ -180,12 +180,15 @@
...
+                        "title": "Kann mein Zertifikat in der App als Berechtigung zum Reisen dienen?",
                         "anchor": "vac_cert_privileges_travel",
                         "active": true,
                         "textblock": [
-                            "Ob Sie mit einem vollständigen Impfschutz beispielsweise reisen dürfen, hängt von den Bestimmungen des jeweiligen Landes ab.",
-                            "Sie können unter anderem dieses <a href='https://reopen.europa.eu/de' target='_blank' rel='noopener noreferrer'>Webseite</a> benutzen, um ihre Reise innerhalb der EU zu planen."
+                            "Der digitale Impfnachweis kann nicht nur in Deutschland genutzt werden. Der in Deutschland ausgestellte digitale Impfnachweis kann auch im EU-Ausland als Nachweis für erfolgte Impfungen gelten, wenn der jeweilige EU-Mitgliedstaat digitale Nachweise zulässt. Dasselbe gilt perspektivisch auch für digitale Genesenenzertifikate und Testzertifikate. Im Allgemeinen prüfen die jeweiligen EU-Mitgliedstaaten mit ihren eigenen Prüf-Apps Ihr Zertifikat, d. h. es ist unerheblich, in welcher Sprache Ihr Zertifikat auf Ihrem Gerät dargestellt wird.",
```
Comment (MikeMcC399) [inline src/data/faq_de.json:187]:
```suggestion
                            "Der digitale Impfnachweis kann nicht nur in Deutschland genutzt werden. Der in Deutschland ausgestellte digitale Impfnachweis kann auch im EU-Ausland als Nachweis für erfolgte Impfungen gelten, wenn der jeweilige EU-Mitgliedstaat digitale Nachweise zulässt. Dasselbe gilt perspektivisch auch für digitale Genesenenzertifikate und Testzertifikate. Im Allgemeinen prüfen die jeweiligen EU-Mitgliedstaaten mit ihren eigenen Prüf-Apps Ihr Zertifikat, d.&nbsp;h. es ist unerheblich, in welcher Sprache Ihr Zertifikat auf Ihrem Gerät dargestellt wird.",
```

**Review by MikeMcC399 — CHANGES_REQUESTED** [review]
Added `&nbsp;` for abbreviation d. h.

**Comment by MikeMcC399** [comment]
@dsarkar 
Is this PR still necessary now that CWA 2.6 has been released with business rules?

https://www.coronawarn.app/en/faq/#cert_eu_travel together with the link to https://reopen.europa.eu/en covers what users need to know about using their certificates for travel purposes.

**Comment by GisoSchroederSAP** [comment]
Good question. 
As far as I know, there is no central place that lists the country-specific "business rules" that are evaluated by the check apps. Content-wise, the REOPEN website may document the national requirements, but the user never will see what rules are set by a country - except, the own certificate failed the validation check - than the user gets an information, what rule was violated.

So, I agree, it's probably not worth putting additional effort in this.

File: src/data/faq_de.json (hunk 2 above)
Comment (dsarkar) [reply src/data/faq_de.json:187]: thanks. I will close this issue, which became stale. A new PR has been submitted. Sorry for the inconvenience .

**Comment by dsarkar** [comment]
@MikeMcC399 @Ein-Tim Thanks for the effort. Another PR has been submitted. One of the requests to enhance this article was to mention that translations of certificated are not necessary, since all EU verifier apps should be able to scan the QR code and show it to the verifier in an appropriate language.

**Pull request closed without merging**